File: app/src/main/java/mattframpton/endclothing_mattframpton/adapter/ProductCatalogueHolder.java
package mattframpton.endclothing_mattframpton.adapter;


import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import mattframpton.endclothing_mattframpton.R;

public class ProductCatalogueHolder extends RecyclerView.ViewHolder {

	TextView productName, productPrice;
	ImageView productImage;

	public ProductCatalogueHolder(View itemView) {
		super(itemView);
		productImage = (ImageView) itemView.findViewById(R.id.productImage);
		productName = (TextView) itemView.findViewById(R.id.productName);
		productPrice = (TextView) itemView.findViewById(R.id.productPrice);
	}
}

File: app/src/main/java/mattframpton/endclothing_mattframpton/adapter/ProductCatalogueHolder.kt
package mattframpton.endclothing_mattframpton.adapter

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import mattframpton.endclothing_mattframpton.R

class ProductCatalogueHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    internal val productName: TextView = itemView.findViewById<View>(R.id.productName) as TextView
    internal val productPrice: TextView = itemView.findViewById<View>(R.id.productPrice) as TextView
    internal val productImage: ImageView = itemView.findViewById<View>(R.id.productImage) as ImageView
}